src/components/Cards/AllCards.jsx

```diff
@@ -3,29 +3,29 @@ import { connect } from 'react-redux';
 import styles from './Cards.module.css';
 
 export class AllCards extends React.Component {
+
     getCardItems() {
         if (this.props.cards) {
             return this.props.cards.map(card => (
-                <img className={styles.allCards} key={card.id} src={require(`../Dashboard/dixit_logo.png`)}></img>
+                <img className={styles.allCards} alt='' key={card.id} src={require(`../Dashboard/dixit_logo.png`)}></img>
```

Comment:
Out of curiosity, does react complain if you don't have an alt on the image?

Reply:
Usually the `alt` property describes the image for accessibility reasons. Although, with this game, I can't image it being very usable for people with sight problems... Think the `alt` prop could kind of break the game too.

Reply:
Yeah I only added it in because React gives a warning if it doesn't have it

Reply:
@tillywoodfield

Reply:
@Ro4052 what do you mean by it could break the game?

Reply:
So the game is based on coming up with a word that represents the image. The `alt` prop also represents the image. We don't want to be giving players hints like that.